lib/block-supports/layout.php

```diff
@@ -172,7 +206,10 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 		$used_layout = $default_layout;
 	}
 
-	$class_name = wp_unique_id( 'wp-container-' );
+	$container_class = wp_unique_id( 'wp-container-' );
+	$class_names     = gutenberg_get_layout_classes( $block['attrs'] );
+	$class_names[]   = $container_class;
+
 	$gap_value  = _wp_array_get( $block, array( 'attrs', 'style', 'spacing', 'blockGap' ) );
```

Comment:
Not caused by this PR, but the linter is complaining about the extra space here. Shall we fix it up in this PR?

```suggestion
	$gap_value = _wp_array_get( $block, array( 'attrs', 'style', 'spacing', 'blockGap' ) );
```